src/main/java/ch/njol/skript/expressions/ExprBlocks.java

```diff
@@ -67,10 +73,12 @@
 	private Expression<Block> end;
 	@Nullable
 	private Expression<Direction> direction;
+	private int pattern;

```

Comment:
a nitpick more than anything but does this need to be an int? I feel like it can just be a boolean considering how it is being used.

Reply:
It could totally be a boolean. I was more future proofing it here. 
I had plans to do some more edits to this class (in another PR), using block cuboids and the such, and figured the pattern int would be useful.

Reply:
Fair enough